client/scss/components/_form-side.scss

```diff
@@ -49,12 +61,38 @@
     }
   }
 
+  &__resize-grip-container {
+    @apply w-absolute w-place-items-center w-hidden sm:w-flex w-z-10;
+
+    height: calc(100% - theme('spacing.8'));
+    inset-inline-start: -21px;
+  }
+
+  &__resize-grip {
```

Comment:
It would be nice if the icon within this was centered and the whole thing was a square. Right now there is more space at the bottom than top:

![resize-grip](https://user-images.githubusercontent.com/877585/196393245-f59899f5-e38c-4108-9ea9-b9b3db4fcd71.png)